atom/browser/api/atom_api_session.cc

```diff
@@ -416,6 +418,16 @@ void Session::DoCacheAction(const net::CompletionCallback& callback) {
                  callback));
 }
 
+void Session::ClearHSTSData(mate::Arguments* args) {
+  base::Closure callback;
+  args->GetNext(&callback);
```

Comment:
this either needs an `if(!` check and dummy callback if one is not supplied or it needs to throw a JS exception if one is not supplied. Otherwise failure to supply a callback will cause a crash